tapsend/send.go

```diff
@@ -172,7 +172,12 @@ type FundingDescriptor struct {
 // TapCommitmentKey is the key that maps to the root commitment for the asset
 // group specified by a recipient descriptor.
 func (r *FundingDescriptor) TapCommitmentKey() [32]byte {
-	return asset.TapCommitmentKey(r.ID, r.GroupKey)
+	assetSpecifier := asset.NewIdSpecifier(r.ID)
```

Comment:
nit: use the `*Optional` constructor here too?

Reply:
I think you commented on outdated code and that I've made the right changes in the latest commits.